OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/WorkshopControllerTests.cs

```diff
@@ -135,6 +135,68 @@ public async Task GetWorkshopById_WhenThereIsNoWorkshopWithId_ShouldReturnNoCont
     }
     #endregion
 
+    #region GetCompetitiveSelectionDescription
+    [Test]
+    public async Task GetCompetitiveSelectionDescription_WhenIdIsValid_ShouldReturnOkResultObject()
+    {
+        // Arrange
+        workshopServiceMoq.Setup(x => x.GetById(It.IsAny<Guid>(), It.IsAny<bool>())).ReturnsAsync(workshop);
+
+        // Act
+        var result = await controller.GetCompetitiveSelectionDescription(workshop.Id) as OkObjectResult;
```

Comment:
Please consider your tests structuring.
`as OkObjectResult` is not part of `Act` section. It should be checked in `Assert`, and it could be checked via

```
        result.Should()
              .BeOfType<OkObjectResult>()
              .Which.StatusCode
              .Should()
              .Be(StatusCodes.Status200OK);
```

Reply:
Improved tests according to your comment.